static/app/views/settings/components/teamSelect/teamSelectForProject.tsx

```diff
@@ -62,7 +62,9 @@ function TeamSelect({
       ? t(
           'This is the last team with access to this project. After removing this team, only organization owners and managers will be able to access the project pages.'
         )
-      : null;
+      : t(
```

Comment:
were we bypassing this in the default case because we pass `bypass={!confirmMessage}` and it used to be null? 😬

Reply:
yup!